web/handlers/admin/setup_test.go

```diff
@@ -56,6 +56,10 @@ type testSession struct {
 	RoomState *roomstate.Manager
 }
 
+func generatePubKey() refs.FeedRef {
+	return refs.FeedRef{Algo: "ed25519", ID: bytes.Repeat([]byte{0}, 32)}
+}
```

Comment:
```suggestion
var pubKeyCount = 0

func generatePubKey() refs.FeedRef {
	pk := refs.FeedRef{Algo: "ed25519", ID: bytes.Repeat([]byte{pubKeyCount}, 32)}
	pubKeyCount++
	return pk
}
```

I like having different keys. This way we can remove it elsewhere were we need that.

Reply:
ah that's an elegant way of solving different keys, thanks!

(i was intending to do some kind random thing, but i didnt really have anything up my sleeve)